ledger/ledger-api-auth/src/main/scala/com/digitalasset/ledger/api/auth/Authorizer.scala

```diff
@@ -179,15 +183,27 @@ final class Authorizer(now: () => Instant, ledgerId: String, participantId: Stri
       },
     )
 
+  private def authenticatedClaimsFromContext(): Try[ClaimSet.Claims] =
+    AuthorizationInterceptor
+      .extractClaimSetFromContext()
+      .fold[Try[ClaimSet.Claims]](Failure(unauthenticated())) {
+        case ClaimSet.Unauthenticated => Failure(unauthenticated())
+        case claims: ClaimSet.Claims => Success(claims)
+      }
+
   private def authorize[Req, Res](call: (Req, ServerCallStreamObserver[Res]) => Unit)(
-      authorized: Claims => Either[AuthorizationError, Unit]
+      authorized: ClaimSet.Claims => Either[AuthorizationError, Unit]
   ): (Req, StreamObserver[Res]) => Unit =
     (request, observer) => {
       val scso = assertServerCall(observer)
-      AuthorizationInterceptor
-        .extractClaimsFromContext()
+      authenticatedClaimsFromContext()
         .fold(
-          observer.onError(_),
+          ex => {
+            logger.debug(
+              s"No authenticated claims found in the request context. Returning UNAUTHENTICATED"
```

Comment:
Thanks for retaining the logs. This is very important for our ability to troubleshoot authorization issues.